.gitignore

```diff
@@ -4,6 +4,8 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
+Gemfile.lock
```

Comment:
Gemfile.lock needs to be committed

e.g. https://depfu.com/blog/2017/01/18/bundler-and-gemfile-best-practices